uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1000.php

```diff
@@ -38,7 +38,7 @@ private function up1000_downloadUKEnglish() {
       mkdir($downloadPath, 0755, true);
     }
 
-    file_put_contents($downloadPath, fopen($localizationURL, 'r'));
+    file_put_contents($downloadPath . 'civicrm.mo', fopen($localizationURL, 'r'));
```

Comment:
Same question, was this broken and if it was, then maybe it could be added to the PR description. Also same question about editing an upgrader when it may have already been run.